php-checks/src/test/java/org/sonar/php/checks/phpunit/NoTestInTestClassCheckTest.java

```diff
@@ -28,4 +28,9 @@ public class NoTestInTestClassCheckTest {
   public void test() {
     CheckVerifier.verify(new NoTestInTestClassCheck(), "phpunit/NoTestInTestClassCheck.php");
   }
+
+  @Test
+  public void test2() {
+    CheckVerifier.verify(new NoTestInTestClassCheck(), "phpunit/NoTestInTestClassCheck2.php");
+  }
```

Comment:
What is the difference from the first test? The name should be more descriptive.